.gitignore

```diff
@@ -2,3 +2,6 @@
 tmp
 .bundle
 vendor
+Gemfile.lock
```

Comment:
An a final annoyance: bundler 1.10.3 adds this at the end of lockfiles:

```
BUNDLED WITH
   1.10.3
```

So it's almost like having a log file checked into the repository and being asked to keep it. It's your choice though, and I respect that.